test/test_tae/test_exec_func.py

```diff
@@ -53,10 +56,22 @@ def test_run(self):
         self.assertEqual(rval[1], 4)
         self.assertGreaterEqual(rval[2], 0.0)
         self.assertEqual(rval[3], {'key': 12345, 'instance': 'test'})
-        
-    
+
```

Comment:
This test should be in the function `test_run_wo_pynisher`, right? It appears that it is already there, so this is a duplicate, isn't it?

Reply:
That is a mistake. It is not supposed to have `use_pynisher` set to false. The reason why I added this test case to both was to test for an exception in the TA in both with & without pynisher runs.